packages/cms-data-sync/src/utils/migration.ts

```diff
@@ -81,13 +81,6 @@ export const migrateFromSquidexToContentfulFactory =
                 return null;
               }
             }
-
-            // if in create mode we could still use
-            // fallbackParseData func so just throw when
-            // not in create mode
-            if (updateEntry || upsertInPlace) {
```

Comment:
I haven't added to the fallbackParseData condition because I don't think there's a harm having a fallback parse when we're using update in place. There's no current use case for that now.